### DISCUSSION TITLE
Support for Kubeflow Pipelines CI/CD on ppc64le without GCP Access

### DISCUSSION THREAD

**Discussion by valen-mascarenhas14** [kubeflow/pipelines#10844] [General]
**Description:**
We are contributing to the Kubeflow Pipelines (KFP) project by enabling CI/CD support for the ppc64le architecture. Here's the [issue](https://github.com/GoogleCloudPlatform/oss-test-infra/issues/1972) we had created.  As part of this effort, we are focusing on the [`kubeflow-pipeline-e2e-test`](https://github.com/GoogleCloudPlatform/oss-test-infra/blob/efbf6f741bfe15de829992d1073b450d3c19cc2e/prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml#L24) job. However, we are facing challenges due to our inability to access Google Cloud Platform (GCP) services from our IBM environment.

**Background:**
GCP is currently integral to the overall CI/CD setup for Kubeflow Pipelines. It provides infrastructure for deployment, authentication, storage, build processes, Kubernetes cluster management, and workload identity. The configuration and scripts leverage various GCP services such as:
**Google Cloud Storage (GCS)** for storing build artifacts and test results.
**Google Cloud Build** for building and testing Docker images.
**Google Kubernetes Engine (GKE)** for managing Kubernetes clusters.
**Google Cloud IAM** for managing permissions and service accounts.
**Google Cloud SDK and CLI** used as base images in many Dockerfiles.

Any guidance or recommendations on how to proceed with alternative cloud providers would be highly appreciated. 

Thanks for your support & guidance .

**Comment by rimolive** [comment]
We have an open issue to move Prow tests to GitHub Actions: https://github.com/kubeflow/pipelines/issues/10744

The reasoning is the same: No one can access this testing environment than the Google members, so the plan is move to an agnostic CI/CD infra and working on some tweaks to generate test artifacts to make things easier to debug.

**Comment by valen-mascarenhas14** [comment]
Thanks @rimolive  
I forked the above repository and planned to run the GitHub Actions workflow locally. However, it still requires access to Google Cloud credentials. The error indicates a failure to obtain the GCP access token. I am observing the same issue in your repository. Are there any plans to address this?
